names.txt

```diff
@@ -6,4 +6,4 @@ zahra
 ALi
 Sina
 arman
-mani sasani 
+mani 
```

Comment:
Thank you for using our Git course, and I hope you have been satisfied with the course so far. 🙏🏻